torch/lib/c10d/TCPStore.cpp

```diff
@@ -273,7 +273,8 @@ TCPStore::TCPStore(
     PortType masterPort,
     int numWorkers,
     bool isServer,
-    const std::chrono::milliseconds& timeout)
+    const std::chrono::milliseconds& timeout,
+    bool wait)
```

Comment:
Nit: may be call expand the name to something more meaningful so that the caller knows what will it wait for with the flag ?

Reply:
Makes sense, will address before landing.